src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/x64/convolution_backprop_data.cpp

```diff
@@ -147,14 +111,49 @@ INSTANTIATE_TEST_SUITE_P(nightly_Deconv_3D_Planar_BF16,
                            ::testing::ValuesIn(Planar_3D_inputs_nightly),
                            ::testing::Values(ElementType::f32),
                            ::testing::ValuesIn(fusingParamsSet),
-                           ::testing::ValuesIn(filterCPUInfoForDevice({conv_gemm_3D})),
+                           ::testing::ValuesIn(filterCPUInfoForDevice({planar_3D})),
                            ::testing::Values(cpu_bf16_plugin_config)),
         DeconvolutionLayerCPUTest::getTestCaseName);
 
-/* ============= Deconvolution (Blocked 2D) ============= */
-const std::vector<DeconvInputData> Blocked_2D_inputs_smoke = {
+/* =============  CVS-140606============= */
+const std::vector<std::vector<size_t>> kernels_Async_Padding_1x1 = {{{1, 1}}};
+const std::vector<std::vector<size_t>> strides_Async_Padding_1x1 = {{{1, 1}}};
+const std::vector<std::vector<ptrdiff_t>> padBegins_Async_Padding_1x1 = {{0, 0}};
+const std::vector<std::vector<ptrdiff_t>> padEnds_Async_Padding_1x1 = {{1, 1}};
+const std::vector<std::vector<size_t>> dilations_Async_Padding_1x1 = {{1, 1}};
+const std::vector<size_t> numOutChannels_Async_Padding_1x1 = {16};
+
+const auto conv2DParams_Async_Padding_1x1 = ::testing::Combine(
+        ::testing::ValuesIn(kernels_Async_Padding_1x1),
+        ::testing::ValuesIn(strides_Async_Padding_1x1),
+        ::testing::ValuesIn(padBegins_Async_Padding_1x1),
+        ::testing::ValuesIn(padEnds_Async_Padding_1x1),
+        ::testing::ValuesIn(dilations_Async_Padding_1x1),
+        ::testing::ValuesIn(numOutChannels_Async_Padding_1x1),
+        ::testing::Values(ov::op::PadType::EXPLICIT),
+        ::testing::ValuesIn(emptyOutputPadding)
+);
+
+const std::vector<DeconvInputData> inputs_numOutChannels_Async_Padding_1x1 = {
+                        DeconvInputData{InputShape{{1, 3, 30, 30}, {{1, 3, 30, 30}}},
+                        ov::test::utils::InputLayerType::CONSTANT,
+                        {}}
+};
+
+INSTANTIATE_TEST_SUITE_P(nightly_Deconv_2D_Async_Padding_1x1, DeconvolutionLayerCPUTest,
+                        ::testing::Combine(
+                                conv2DParams_Async_Padding_1x1,
+                                ::testing::ValuesIn(inputs_numOutChannels_Async_Padding_1x1),
+                                ::testing::Values(ElementType::f32),
+                                ::testing::ValuesIn({emptyFusingSpec}),
+                                ::testing::ValuesIn(filterCPUInfoForDevice({conv_avx512_2D_1x1_nspc_brgconv})),
```

Comment:
May be add `CPUSpecificParams{{nhwc}, {nhwc}, {""}, "any_type"};` to check that we have save `nhwc` path for such a parameters set?

Reply:
can't work. As I said in the teams, the "ref_deconvolution_fwd_t" would also return backward conv implementation with brgconv after specifying nhwc layout. brg conv is removed by buggy code in deconv for this case. So deconv OP would raise exception because of no supported primitive for this case. So current WA is  to skip this test case.